### PR TITLE
Add vmware-vsphere-csi-driver{-operator} repositories

### PR DESCRIPTION
**Pull request by bertinatto**
Only run unit tests for now.

CC @openshift/storage

### CODE REVIEW

File: ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+base_images:
```
Comment (jsafrane) [inline ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml]: Why is openshift-azure-disk-csi-driver-operator-master.yaml  file created?

File: ci-operator/config/openshift/vmware-vsphere-csi-driver/OWNERS [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+options: {}
```
Comment (jsafrane) [inline ci-operator/config/openshift/vmware-vsphere-csi-driver/OWNERS:4]: Empty OWNERs  is suspicious

File: ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml (hunk 1 above)
Comment (jsafrane) [reply ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml]: It's fixed now

File: ci-operator/config/openshift/vmware-vsphere-csi-driver/OWNERS (hunk 2 above)
Comment (bertinatto) [reply ci-operator/config/openshift/vmware-vsphere-csi-driver/OWNERS:10]: Fixed

File: ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-vmware-vsphere-csi-driver-operator-master.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.8"
+  namespace: ocp
+tests:
+- as: unit
+  commands: "true"
```
Comment (bertinatto) [inline ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-vmware-vsphere-csi-driver-operator-master.yaml:26]: Will update this to `commands: make test-unit` once we're able to merge the initial code.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**